src/WebJobs.Extensions.DurableTask/Options/HttpOptions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Options
+{
+    /// <summary>
+    /// Used for Durable HTTP functionality.
+    /// </summary>
+    public class HttpOptions
+    {
+        /// <summary>
+        /// Reserved name to know when a TaskActivity should be an HTTP activity.
+        /// </summary>
+        internal const string HttpTaskActivityReservedName = "Durable:Http:Async:Activity:Function";
+
+        /// <summary>
+        /// Time between the async http requests.
```

Comment:
I would say "Gets or sets the default number of milliseconds between async HTTP status poll requests."